airflow/utils/db.py

```diff
@@ -501,3 +501,13 @@ def resetdb():
     Base.metadata.drop_all(connection)  # pylint: disable=no-member
 
     initdb()
+
+
+@provide_session
+def check(session=None):
+    """
+    Checks if the database works.
+    :param session: session of the sqlalchemy
+    """
+    session.execute('select 1 as is_alive;')
+    log.info("Connection successful.")
```

Comment:
I think the `health` endpoint I mentioned above would be more useful as it checks if the Scheduler can connect to the metadata DB. `session.execute('select 1 as is_alive;')` would check if the machine where the cli is run can execute that query on the Metadb -> i.e. it will check connection between Machine running CLI with DB

Reply:
WDYT? 
If it is just for CI, how about just using curl to access the endpoint?

Reply:
Again - we do not have neither scheduler nor webserver nor any of the workers. It's just checking if the environment we have is "sane" - i.e. if the databases are there and can be queried, if all the integration for CI tests are reachable at the destination endpoints. 

We explicitly only check database availability before we start tests.